tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/DbScopeFactory.cs

```diff
@@ -68,14 +70,35 @@ private static Scope CreateDbCommandScope(Tracer tracer, IDbCommand command, Int
                 if (tracer.Settings.DbmPropagationMode != DbmPropagationLevel.Disabled
                     && command.CommandType != CommandType.StoredProcedure)
                 {
-                    var propagatedCommand = DatabaseMonitoringPropagator.PropagateSpanData(tracer.Settings.DbmPropagationMode, tracer.DefaultServiceName, scope.Span, integrationId, out var traceParentInjected);
-
-                    if (propagatedCommand != string.Empty)
+                    var alreadyInjected = commandText.StartsWith(DatabaseMonitoringPropagator.DbmPrefix);
```

Comment:
_Removing_ the comment is _exactly_ what I was trying to avoid, as that essentially involves parsing the `commandText` 😄 If we _do_ want to do that, then I wonder if we should do something else entirely tbh. Like storing the original `commandText` somewhere (or something). If we did that we could "fix" the `full` version too, but it's a lot more work...

Personally I'm inclined to not do that for now, as this is only a "theoretical" issue currently...